components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -619,7 +620,7 @@ - (UIOffset)floatingPlaceholderOffset {
   // Offsets needed due to transform working on normal (0.5,0.5) anchor point.
   // Why no anchor point of (0,0)? Because autolayout doesn't play well with anchor points.
   vertical -= self.textInput.placeholderLabel.font.lineHeight *
-              (1 - (CGFloat)self.floatingPlaceholderScale.floatValue) * .5f;
+              (1 - (CGFloat)self.floatingPlaceholderScale.floatValue) * (CGFloat).5;
```

Comment:
tip: for consistency, let's make it `(CGFloat)0.5`

Reply:
Done throughout.